planning/static_centerline_generator/include/static_centerline_generator/centerline_source/optimization_trajectory_based_centerline.hpp

```diff
@@ -12,32 +12,43 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#ifndef STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_  // NOLINT
-#define STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_  // NOLINT
+#ifndef STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_
+STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_  // NOLINT
+#define STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_
+  STATIC_CENTERLINE_GENERATOR__CENTERLINE_SOURCE__OPTIMIZATION_TRAJECTORY_BASED_CENTERLINE_HPP_  // NOLINT
 
 #include "rclcpp/rclcpp.hpp"
+#include "route_handler/route_handler.hpp"
 #include "static_centerline_generator/type_alias.hpp"
 
+#include "autoware_auto_planning_msgs/msg/path.hpp"
+#include "autoware_auto_planning_msgs/msg/path_with_lane_id.hpp"
+#include "autoware_auto_planning_msgs/msg/trajectory_point.hpp"
+
 #include <vector>
 
-namespace static_centerline_generator
-{
-class OptimizationTrajectoryBasedCenterline
+  namespace autoware::static_centerline_generator
 {
-public:
-  OptimizationTrajectoryBasedCenterline() = default;
-  explicit OptimizationTrajectoryBasedCenterline(rclcpp::Node & node);
-  std::vector<TrajectoryPoint> generate_centerline_with_optimization(
-    rclcpp::Node & node, const RouteHandler & route_handler,
-    const std::vector<lanelet::Id> & route_lane_ids);
+  using ::autoware_auto_planning_msgs::msg::Path;
+  using ::autoware_auto_planning_msgs::msg::PathWithLaneId;
+  using ::autoware_auto_planning_msgs::msg::TrajectoryPoint;
+  using ::route_handler::RouteHandler;
```

Comment:
type_alias.hpp file is used for the list of `using`.
Any reason to add this list of `using` here?

Reply:
Yes, but I want to make the `using` clauses unified by adding them only in `type_alias.hpp`.
If you highly recommend adding them outside `type_alias.hpp`, we'd better remove `type_alias.hpp` to make them unified.

Reply:
In the latest iteration of this PR, type_alias is used instead of locally defining these.